src/views/swap/steps.js

```diff
@@ -135,33 +140,22 @@ const StyledStepTwo = ({ classes, swapInfo, swapResponse }) => (
       <QrCode size={300} link={swapResponse.bip21} />
     </View>
     <View className={classes.info}>
-      <p
-        style={{
-          fontSize: '30px',
-        }}
-      >
-        Send {}
+      <p className={classes.text}>
+        Send{' '}
         <b>
-          {toWholeCoins(swapResponse.expectedAmount)} {swapInfo.base}
+          {' '}
+          {toWholeCoins(swapResponse.expectedAmount)} {swapInfo.base}{' '}
         </b>{' '}
-        to:
+        <br />
+        on <b>{getCurrencyName(swapInfo.base)}</b> <br />
+        blockchain address:
       </p>
-      <p className={classes.address} id="copy-address">
+      <p id="copy-address" className={classes.address}>
```

Comment:
I renamed the id of the element to copy to just `copy` and moved the `copyToClipBoard` method into the utils folder in the [reverse swap PR](https://github.com/BoltzExchange/boltz-frontend/pull/49/files#diff-13f3ac8aec59f01de2a366e701202512R81).

If it gets merged first keep that in mind when resolving the conflicts

Reply:
Yeah, I noticed, thanks for the heads up tho.